stock_location_area_management/README.rst

```diff
@@ -1,50 +1,56 @@
-Features
-========
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-This module was written to enter a location's area based on different units of measure
-The information is dependant on the different locations entered
+==============================
+Stock Location Area Management
+==============================
 
-Screenshots
-===========
-
-... next step (after development)
-
-Installation
-============
-
-To install this module, you just need to select the module and insure yourself dependencies are available
-
-Configuration
-=============
-
-No particular configuration to use this module
+This module was written to enter a location's area based on different units of measure.
```

Comment:
This module allows to enter a location's area based on different units of measure.